deploy-jenkins/templates/jenkins-persistent-template.j2

```diff
@@ -274,26 +274,26 @@
       "name": "MEMORY_LIMIT",
       "displayName": "Memory Limit",
       "description": "Maximum amount of memory the container can use.",
-      "value": "3Gi"
+      "value": "{{ master_memory_limit }}"
     },
     {
       "name": "VOLUME_CAPACITY",
       "displayName": "Volume Capacity",
       "description": "Volume space available for data, e.g. 512Mi, 2Gi.",
-      "value": "40Gi",
+      "value": "{{ master_volume_capacity }}",
       "required": true
     },
     {
       "name": "JENKINS_IMAGE_NAME",
       "displayName": "Jenkins Image Name",
       "description": "Name of the Image to be used in the Jenkins Container",
-      "value": "openshift/jenkins-2-centos7"
+      "value": "{{ jenkins_image_name }}"
```

Comment:
I don't think we should allow the user to override this value as this in the image and version of jenkins that we tested everything against.